.github/workflows/windows-msvc-cuda.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         config:
         - {version: "latest", name: "cuda-latest/release/shared", "mixed": "ON"}
     name: msvc/${{ matrix.config.name }} (only compile)
-    runs-on: [windows-latest]
+    runs-on: [windows-2019]
```

Comment:
I guess a more permanent fix would be nice.

Reply:
Agreed, but I've seen other pipelines fail with the same problem, e.g. https://github.com/Jimver/cuda-toolkit/actions/runs/8767123552/job/24060026380#step:11:101 so there is something fishy going on with Github's Windows setup in general.